### PR TITLE
Hardening, batch II

### PR DESCRIPTION
**Pull request by wouter-admiraal-sonarsource**
This is required for https://github.com/SonarSource/sonar-enterprise/pull/3815.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,13 @@
 
 ## Unreleased
 
+- SONAR-12056 Indicate required fields for forms 
+- SONAR-11959 Colorblind-friendly coverage treemaps
+
 ## 1.0.31
 
+- SONAR-12180 Add a ARIA role to all global success and error messages
+
```
Comment (jeremy-davis-sonarsource) [inline CHANGELOG.md:11]: What happened here?

File: components/charts/ColorGradientLegend.tsx [hunk 2]
```diff
@@ -49,45 +51,81 @@ export default function ColorGradientLegend({
...
+                    x2={i}
+                    y2="30"
+                    style={{ stroke: colors.gray71, strokeWidth: NA_SPACING }}
+                  />
+                  <line
+                    x1={i + NA_SPACING}
+                    y1="0"
+                    x2={i + NA_SPACING}
+                    y2="30"
+                    style={{ stroke: colors.gray60, strokeWidth: NA_SPACING }}
```
Comment (jeremy-davis-sonarsource) [inline components/charts/ColorGradientLegend.tsx:84]: NP: Are these values really tied to the ones below? I mean, hey are the same value, but should they be the same constant?

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM, WFV

File: CHANGELOG.md (hunk 1 above)
Comment (wouter-admiraal-sonarsource) [reply CHANGELOG.md:11]: I forgot to update the Changelog last time 🤷

File: components/charts/ColorGradientLegend.tsx (hunk 2 above)
Comment (wouter-admiraal-sonarsource) [reply components/charts/ColorGradientLegend.tsx:84]: You mean, if it makes sense to have `strokeWidth: NA_SPACING` being tied to `i + NA_SPACING`? If so, yes, it allows us to keep an even pattern of 4px wide strokes, spaced 0px apart.

Comment (jeremy-davis-sonarsource) [reply components/charts/ColorGradientLegend.tsx:84]: I meant tied to the `rect` and `text` below, but yeah, I see the pattern is actually used by the `rect`, so nevermind :+1:

**Pull request merged**